spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -33,14 +32,14 @@ MicroProfile applications that make use of APIs or versions not mentioned below
  - <<jakarta-jsonb, JSON-B (Jakarta JSON Binding) 1.0>>
  - <<jakarta-jsonp, JSON-P (Jakarta JSON Processing) 1.1>>
  - <<jakarta-annotations, Jakarta Annotations 1.3>>
- - <<mp-config, Eclipse MicroProfile Config 1.4>>
- - <<mp-fault-tolerance, Eclipse MicroProfile Fault Tolerance 2.1>>
- - <<mp-health-check, Eclipse MicroProfile Health 2.2>>
- - <<mp-jwt-auth, Eclipse MicroProfile JWT Authentication 1.1>>
- - <<mp-metrics, Eclipse MicroProfile Metrics 2.3>>
- - <<mp-openapi, Eclipse MicroProfile OpenAPI 1.1>>
- - <<mp-opentracing, Eclipse MicroProfile OpenTracing 1.3>>
- - <<mp-rest-client, Eclipse MicroProfile Rest Client 1.4>>
+ - <<mp-config, MicroProfile Config 2.0-RC1>>
```

Comment:
Since this is the Platform RCx release, I would think this document should reference the real expected releases of the components (and not reference the RCx releases.

Reply:
Fixed